src/test/java/io/xstefank/wildfly/bot/PRDescriptionCheckTest.java

```diff
@@ -34,11 +41,6 @@ void setUp() {
                       regexes:
                         - pattern: "https://issues.redhat.com/browse/WFLY-\\\\d+"
                           message: "The PR description must contain a link to the JIRA issue"
-                        - pattern: "JIRA:\\\\s+https://issues.redhat.com/browse/WFLY-\\\\d+"
-                    title:
-                      enabled: false
-                    commit:
-                      enabled: false
```

Comment:
L#37 was removed because it didn't make sense to me to have 2 almost identical regexes. Now I see that the whole regexes format group is meant to pass all regexes, not just one of them. Is that intended? That would change/add some tests

Also, the rest was removed as title and commit are both enabled by default and I believe the issue #101 stated that checks should be left enabled, unless it was desired to disable them

Reply:
We should have tests that validate multiple regex lines matching. If not in this PR, create a follow up issue.